Themes/default/webfonts/index.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+// Try to handle it with the upper level index.php. (it should know what to do.)
+if (file_exists(dirname(__DIR__) . '/index.php'))
+	include (dirname(__DIR__) . '/index.php');
+else
+	exit;
```

Comment:
```suggestion
// Try to handle it with the upper level index.php. (it should know what to do.)
if (file_exists(dirname(__DIR__) . '/index.php')) {
	include dirname(__DIR__) . '/index.php';
} else {
	exit;
}
```

That will make the build.php CI action happy again.